onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -563,6 +564,47 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
   }
 }
 
+template <typename T>
+static void FillTensorDataTyped(Ort::Value& tensor, size_t count, T value = T{}) {
+  T* data = tensor.GetTensorMutableData<T>();
+  std::fill_n(data, count, value);
+}
```

Comment:
I would say simply do memset(tensor.MutableDataRaw(), 0, tensor.SizeInBytes()) and no switch or templates.
Another idea is to emulate other frameworks and add SetToZero() method to Tensor class?

Reply:
is there an equivalent to `tensor.SizeInBytes()` in the C++ API?

Reply:
No. Our API needs work.

Reply:
looks like this file is using the C++ API and not the internal classes. was hoping to keep this PR's fix local to onnxruntime_perf_test and not have to add new public APIs

Reply:
ok